src/lib/RoomsList/RoomsList.vue

```diff
@@ -137,15 +148,20 @@ export default {
     isMobile: { type: Boolean, required: true },
     rooms: { type: Array, required: true },
     archivedRooms: { type: Array, required: true },
-    customSearchRooms: { type: Array, required: false, default: () => []},
+    customSearchRooms: { type: Array, required: false, default: () => [] },
+    unreadRooms: { type: Array, required: true },
+    groupRooms: { type: Array, required: true },
     loadingRooms: { type: Boolean, required: true },
     roomsLoaded: { type: Boolean, required: true },
     room: { type: Object, required: true },
     customSearchRoomEnabled: { type: [Boolean, String], default: false },
     roomActions: { type: Array, required: true },
     scrollDistance: { type: Number, required: true },
     call: { type: Object, required: true },
-    showArchivedRooms: { type: Boolean, required: true, default: false}
+    showArchivedRooms: { type: Boolean, required: true, default: false },
+    showUnreadRooms: { type: Boolean, required: true, default: false },
+    showGroupRooms: { type: Boolean, required: true, default: false },
+    roomFilterSelected: { type: String, required: true }
```

Comment:
```suggestion
    roomsFilter: { type: String, required: true }
```

Acho que poderíamos só chamar de `roomsFilter`, ou `activeRoomsFilter` como estamos comentando no PR. Fica um pouco mais claro. Não podemos remover o `showArchivedRooms`, `showUnreadRooms` e afins por algo como `roomsFilter == RoomFilterTypes.UNDEAD`, etc?

Reply:
Sim, consegui remover todos o `showALGUMACOISA` e to fazendo a lógica com uma única váriável que controla qual é o filtro ativo. Obrigado por pontuar essa melhoria.